src/test/java/org/opentripplanner/routing/fares/ATLFareServiceTest.java

```diff
@@ -0,0 +1,315 @@
+                getRide(MARTA_AGENCY_ID, 4)
+        );
+        calculateFare(rides, Fare.FareType.electronicRegular, DEFAULT_RIDE_PRICE_IN_CENTS * 2);
+
+        rides = Arrays.asList(
+            getRide(MARTA_AGENCY_ID, 0),
+            getRide(MARTA_AGENCY_ID, 1),
+            getRide(MARTA_AGENCY_ID, 2),
+            getRide(GCT_AGENCY_ID, 3),
+            getRide(GCT_AGENCY_ID, 4)
+        );
+        calculateFare(rides, Fare.FareType.electronicRegular, DEFAULT_RIDE_PRICE_IN_CENTS);
+    }
+
+    @Test
+    public void expiredTransfer() {
+        List<Ride> rides = Arrays.asList(
+            getRide(MARTA_AGENCY_ID, 0),
+            getRide(MARTA_AGENCY_ID, 1),
+            getRide(MARTA_AGENCY_ID, 181),
+            getRide(MARTA_AGENCY_ID, 179)
+        );
+        calculateFare(rides, Fare.FareType.electronicRegular, DEFAULT_RIDE_PRICE_IN_CENTS * 2);
+
+        rides = Arrays.asList(
+            getRide(MARTA_AGENCY_ID, 0),
+            getRide(GCT_AGENCY_ID, 1),
+            getRide(GCT_AGENCY_ID, 181),
+            getRide(MARTA_AGENCY_ID, 181 + 178),
+            getRide(MARTA_AGENCY_ID, 181 + 179)
+        );
+        calculateFare(rides, Fare.FareType.electronicRegular, DEFAULT_RIDE_PRICE_IN_CENTS * 2);
+    }
+
+    @Test
+    public void useStreetcar() {
+        final float STREETCAR_PRICE = DEFAULT_RIDE_PRICE_IN_CENTS-100f;
+        List<Ride> rides = Arrays.asList(
+            getRide(MARTA_AGENCY_ID, 0),
+            getRide(MARTA_AGENCY_ID, "atlsc", 1 ),
+            getRide(MARTA_AGENCY_ID, 2),
+            getRide(MARTA_AGENCY_ID, 3),
+            getRide(MARTA_AGENCY_ID, 4)
+        );
+        calculateFare(rides, Fare.FareType.electronicRegular, DEFAULT_RIDE_PRICE_IN_CENTS + STREETCAR_PRICE);
+
+        rides = Arrays.asList(
+            getRide(COBB_AGENCY_ID, 0),
+            getRide(MARTA_AGENCY_ID, "atlsc", 1),
+            getRide(COBB_AGENCY_ID, "101", 2)
+        );
+        calculateFare(
+            rides,
+            Fare.FareType.electronicRegular,
+            DEFAULT_RIDE_PRICE_IN_CENTS + 100 + STREETCAR_PRICE);
+    }
+
+    /**
+     * These tests are designed to specifically validate ATL fares. Since these fares are hard-coded, it is acceptable
+     * to make direct calls to the Orca fare service with predefined routes. Where the default fare is applied a test
+     * substitute {@link OrcaFareServiceImpl#DEFAULT_TEST_RIDE_PRICE} is used. This will be the same for all cash fare
+     * types.
+     */
+    private static void calculateFare(List<Ride> rides,
+                                      Fare.FareType fareType,
+                                      float expectedFareInCents
+    ) {
+        Fare fare = new Fare();
+        atlFareService.populateFare(fare, null, fareType, rides, null);
+        Assertions.assertEquals(expectedFareInCents, fare.getFare(fareType).getCents());
+    }
+
+
+    private static Ride getRide(String agencyId, long startTimeMins) {
```

Comment:
As Binh already pointed out: please remove the unused methods here. Intellij will tell you which ones they are.

Reply:
Oops! I had this sitting in my git staged but hadn't committed it.